pkg/discovery/k8s_discovery_client.go

```diff
@@ -280,6 +281,15 @@ func (dc *K8sDiscoveryClient) discoverWithNewFramework(targetID string) ([]*prot
 		entityDTOs = append(entityDTOs, serviceDtos...)
 	}
 
+	// 4. build entityDTOs for volumes
+	volumeEntityDTOBuilder := dtofactory.NewVolumeEntityDTOBuilder(podVolumeMetrics)
+	volumeEntityDTOs, err := volumeEntityDTOBuilder.BuildEntityDTOs(clusterSummary.VolumeToPodsMap)
+	if err != nil {
+		glog.Errorf("Error while creating volume entityDTOs: %v", err)
+	}
+	glog.V(3).Infof("There are %d Storage Volume entityDTOs.", len(volumeEntityDTOs))
+	entityDTOs = append(entityDTOs, volumeEntityDTOs...)
```

Comment:
nit: V(2) looks better as this is an important message which can be shown by default and would be good to put these under else block (err == nil)

Reply:
Agree, Updated!